src/main/java/org/broadinstitute/consent/http/db/DataAccessRequestDAO.java

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.consent.http.db;
+
+import org.broadinstitute.consent.http.db.mapper.DataAccessRequestMapper;
+import org.broadinstitute.consent.http.models.DataAccessRequest;
+import org.broadinstitute.consent.http.models.DataAccessRequestData;
+import org.jdbi.v3.json.Json;
+import org.jdbi.v3.json.internal.JsonArgumentFactory;
+import org.jdbi.v3.sqlobject.config.RegisterArgumentFactory;
+import org.jdbi.v3.sqlobject.config.RegisterRowMapper;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.customizer.BindList;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+import org.jdbi.v3.sqlobject.transaction.Transactional;
+
+import java.util.List;
+
+@RegisterRowMapper(DataAccessRequestMapper.class)
+public interface DataAccessRequestDAO extends Transactional<DataAccessRequestDAO> {
+
+    @SqlQuery("select * from data_access_request")
```

Comment:
nit: use standard SQL capitalization, e.g. `SELECT * FROM data_access_request` - would help with readability

Reply:
Will do, thank you!